src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,59 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetCollectionTest {
+
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@DisplayName("size()메소드를 통해 크기를 확인한다")
+	@Test
+	void checkSize() {
+	    // given
+		int numbersSize = numbers.size();
+
+	    // when
+	    // then
+		assertThat(numbersSize).isEqualTo(3);
+	}
+
+	@DisplayName("contains()메소드를 통해 값이 존재하는지 확인한다")
+	@ParameterizedTest
+	@ValueSource(ints = {1, 2, 3})
+	void checkContains(int targetNumbers) {
+	    // given
+	    // when
+	    // then
+		assertThat(numbers.contains(targetNumbers))
+				.isTrue();
+	}
```

Comment:
```suggestion
	@ValueSource(ints = {1, 2, 3})
	void checkContains(int targetNumber) {
		assertThat(numbers).contains(targetNumber);
	}
```

이것 또한 마찬가지로 `contains`를 활용하면 검증을 쉽게 할 수 있어요
그리고 1, 2, 3 이란 값이 하나씩 들어오기 때문에 targetNumbers라는 복수형 대신 targetNumber가 올바른 표현 같습니다.